controllers/storagecluster/reconcile.go

```diff
@@ -144,6 +146,8 @@ func (r *StorageClusterReconciler) Reconcile(ctx context.Context, request reconc
 	defer func() { r.Log = prevLogger }()
 	r.Log = r.Log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 
+	condition, _ := NewUpgradeable(r.Client)
```

Comment:
Ignoring this error seems dangerous. If this fails something has gone horribly wrong. :)

Reply:
updated